src/export/StructureDefinitionExporter.ts

```diff
@@ -202,6 +197,34 @@ export class StructureDefinitionExporter {
     this.setRules(structDef, fshDefinition);
     this.validateStructureDefinition(structDef);
 
-    this.structDefs.push(structDef);
+    if (structDef.type === 'Extension') {
+      this.extensionDefs.push(structDef);
+    } else {
+      this.profileDefs.push(structDef);
+    }
+  }
+
+  /**
+   * Exports Profiles and Extensions to StructureDefinitions
+   * @returns {AllStructureDefinitions}
+   */
+  export(): AllStructureDefinitions {
+    [...this.tank.getAllProfiles(), ...this.tank.getAllExtensions()].forEach(sd => {
```

Comment:
Similar to comment on line 161. Maybe adding another function to `FSHTank` called `getAllStructureDefinitions()`.

Reply:
I like this idea as well, similar to the previous one.